release-notes/release-2.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: February 13, 2024
+---
+
+# Features
+- PHP Azure Build Pipelines
+- [RUM Mobile - React Native](https://docs.middleware.io/rum/rum-react)
+- Spend Analysis charts
+- Trace correlation drill down from specific log messages
+- Price calculator that analyzes your bill based on the usage of different products
+- Sort and group attributes of log-collected data
+
+# Changes
+- Billing tab now displays information about plan name, billing date, billing address, payment details and option to update billing information
+- Invoice tab now displays all past invoices with invoice date, paid date, and the option to download invoices
+- New default dashboard for Kubernetes with pod and cluster overview metrics
```

Comment:
This is two things mashed into one. First, "New default dashboards for all Kuberenetes components" which should  be here in changes and proably worth a screenshot.

The other part of this is the Pod overview which should be a new feature, "Pod overview using a honeycomb style visualization" also shows a screenshot